docs/ref/files/storage.txt

```diff
@@ -11,7 +11,18 @@ Django provides convenient ways to access the default storage class:
 
 .. data:: storages
 
-    Storage instances as defined by :setting:`STORAGES`.
+    A dictionary-like object that allows you to retrieve a storage instance
+    using its alias as defined by :setting:`STORAGES`.
+
+    ``storages`` has a dictionary attribute ``backends``, which at startup is
+    the raw value provided to :setting:`STORAGES`.
```

Comment:
I know I'm the non-native English speaker in this reviewer set but this reads odd to me, I would suggest:
```suggestion
    the raw value provided in :setting:`STORAGES`.
```

Reply:
🤷‍♀️ either work for me. As you think is clearer 👍